_posts/0001-01-01-faq.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
I notice that the pages are in the `_posts` folder and sort of named as though they were blog posts. Is it difficult to organize them in a `_pages` folder and drop the date syntax? Similar to this:

http://www.taniarascia.com/make-a-static-website-with-jekyll/
https://github.com/taniarascia/startjekyll